proto/osmosis/lockup/tx.proto

```diff
@@ -49,3 +51,22 @@ message MsgBeginUnlocking {
   ];
 }
 message MsgBeginUnlockingResponse { bool success = 1; }
+
+// MsgExtendLockup extends the existing lockup's duration.
+// The new duration is longer than the original. 
+message MsgExtendLockup {
+  string owner = 1 [ (gogoproto.moretags) = "yaml:\"owner\"" ];
+  uint64 ID = 2;
+
+  // duration to be set. fails if lower than the current duration, or is unlocking
+  google.protobuf.Duration duration = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdduration) = true,
+    (gogoproto.jsontag) = "duration,omitempty",
+    (gogoproto.moretags) = "yaml:\"duration\""
+  ];
+
+  // extend for other edit, e.g. cancel unlocking
+}
+
+message MsgExtendLockupResponse { bool success = 1; }
```

Comment:
Should we remove this response field? (Can be done in a second PR)

Not sure what this would communicate relative to the general txs success / failure